jwt/exceptions.py

```diff
@@ -1,4 +1,11 @@
-class InvalidTokenError(Exception):
+class PyJwtError(Exception):
```

Comment:
Small nitpick, can we rename this to `PyJWTError`? Also, we'll need to expose this in `pyjwt/__init__.py`.

Reply:
for you? of course